news/33.feature

```diff
@@ -0,0 +1,2 @@
+Add translations to locales
```

Comment:
@aormazabal Please add new languages here.

You can do this on a branch in the repository, rather than a fork.

Reply:
And please renamed the file to `35.feature` since the PR is 35.

Reply:
@wesleybl Either the PR number (35) or the issue number (33) is okay here, since the PR fixes issue #33

Reply:
@davisagli well the 33 is another PR. But OK.

Reply:
@wesleybl Oops, you are right.